lib/widgets/explore/product_item.dart

```diff
@@ -1,15 +1,57 @@
 import 'package:flutter/material.dart';
 
 class ProductItem extends StatelessWidget {
-  String title;
+  // TODO: Image;
+  final String title;
+  final String brand;
+  final String size; //? Should this be int or String?
+  //? Sizes could be either "small" or "36"
 
-  ProductItem(this.title);
+  ProductItem(
+      {@required this.title, @required this.brand, @required this.size});
 
   @override
   Widget build(BuildContext context) {
-    // TODO: Implement.
-    return ListTile(
-      title: Text(title),
+    return GestureDetector(
+      onTap: () {
+        //TODO: Send user to selected product.
+        print(title);
+      },
+      child: Card(
+        //* elevation = 0 makes the card appear more "futuristic" and clean
```

Comment:
Does the * in this comment mean anything? If not remove it. If it fits place it on the same line as elevation and finish the sentence with a dot.